src/shared/Gitee/GiteeConstants.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+
+namespace Gitee
+{
+    public static class GiteeConstants
+    {
+        public static readonly Uri GiteeDotCom = new Uri("https://gitee.com");
+
+        public const string DefaultAuthenticationHelper = "Gitee.UI";
+
+        // Owned by https://gitee.com/GitCredentialManager/git-credential-manager/issues
+        public const string OAuthClientId = "09776da975878a86ed655d5ada62a5d4e6faf8e89514ec982794eb61f4092d01";
+        public const string OAuthClientSecret = "1d52e7bb3c53b396ad3c4e256ca1eaff27df20a782cd4f1f3dfefa32f44bafc9";
+
+        public static readonly Uri OAuthRedirectUri = new Uri("http://127.0.0.1:52344/");
```

Comment:
Explicit port shouldn't be necessary, assuming the server correctly implements OAuth [RFC 8252](https://datatracker.ietf.org/doc/html/rfc8252#section-7.3)

> The authorization server MUST allow any port to be specified at the time of the request for loopback IP redirect URIs

Have you reported a bug to Gitee?

Reply:
Already reported! After the Chinese New Year holiday, I will keep in touch with Gitee.Com .

Reply:
Brilliant, thanks for reporting

Reply:
@maikebing any news?